### ISSUE TITLE
Extend the data shown in the duplicate person popup [1]

### ISSUE THREAD

**Issue by Marko-ilmkreis** [hzi-braunschweig/SORMAS-Project#4003]
**TODO:** https://github.com/hzi-braunschweig/SORMAS-Project/issues/4003#issuecomment-790177585

### Feature Description

If you enter or import a duplicate, a pop-up comes with the data of both persons. This  shows for the new person the date of birth only and for the person in Sormas the age:

![grafik](https://user-images.githubusercontent.com/74791118/104581893-7df4a900-565f-11eb-89d9-3a615cd1c9c7.png)

It would be better and make the person comparable if either both or only the date of birth would shown for both entries. Only shown the age doesn't help.

**Comment by bernardsilenou** [comment]
So we only show age if the person is sormas has only age registered.

**Comment by Marko-ilmkreis** [comment]
@bernardsilenou I think there are dates entered in the case, as we never enter the age only. But I will check it tomorrow and provide screenshots.

**Comment by bernardsilenou** [comment]
@Marko-ilmkreis In Germany, we always have birth date but in other countries it is common to have just age only.

**Comment by Marko-ilmkreis** [comment]
@bernardsilenou Thats right. I also have customers from other countries and in the past the births were not registered. But as we have entered the dob in any case, in the duplicate pop-up it must shown to compare the cases.

**Comment by Candice-Louw** [comment]
Further support to include the date of birth, please (from CH).

Additionally, including the Person-ID and telphone number has also been requested as these are likely to be more unique. @Marko-ilmkreis @bernardsilenou - would these make sense to you too, and could they be included in this Issue, or should I create a separate Issue for that for CH servers only?

**Comment by bernardsilenou** [comment]
@Candice-Louw Excellent point!
 I think telephone number sound good to add.
Since the Person-ID is not often in the data being imported, then I think it may not be helpful to add.
These are minor changes, so we can handle all the in this ticket I suppose.
If there is no further refinement to make, we can then edit the main issue and summarize the main point

**Comment by Marko-ilmkreis** [comment]
@Candice-Louw I agree  also with the phone number. As @bernardsilenou written the Person-ID is usually not imported, but in my opinion it could be good to see when the duplicate in the database was created. So the user could see if its maybe a duplicate which was entered the same day by another colleague.

**Comment by bernardsilenou** [comment]
@Marko-ilmkreis Fro cases, yes we show the date, but for person we do not because the person can stay in the system as long as the law of the country permits. 
The date that the person was created may not give the information that the user want because:
assume a case created was created today, then the person is created today, if after a 3 months or a year and just before the case is deleted the person become a new case (possibly of another disease or the same) or a contact or event part, then when creating the new contact in future, the date of the person would show the date of the initial case.
Users who do not understand the whole logic may think it is another person.

**Comment by Marko-ilmkreis** [comment]
@bernardsilenou Ok, that make sense not to show for the person itself. Thanks for the explanation!

**Comment by kwa20** [comment]
@bernardsilenou Considering the birthdate I also noticed that the system does not allow any discrepancy when the birthdate is provided and does not suggest that there are any duplicates. We should probably allow some variance to account for selection errors.

**Comment by bernardsilenou** [comment]
@kwa20 Many people can have the same address and a person can also have many addresses.
We can add address but i think this may not increase the sensitivity of the system.
If we can find a variable that is unique to a person and is documented most of the time, then we can add. 
The window is almost full.
For adding some variability/ tolerance to birth date or age is also problematic because we want the system to be more conservative ie it should only suggest duplicates to the user when they are true duplicates. It is better to not suggest a duplicate rather than suggesting false duplicate. 
We can easily merge the same person that was not recorded twice or more as a duplicate, but we cannot "un-merge" retrospectively after a wrong merge was done.
One option is to add the address and other variables to the "merge duplicate function" instead

**Comment by kwa20** [comment]
@bernardsilenou I thought of the address more of an indicator for the user, not necessarily as a criteria for the system to recognize duplicates. The address might just help users to make a decision. In regards of merging, we do not have a way to merge persons yet, only cases, therefore, it's not as easily resolved if duplicates are not detected. I guess this will eventually be resolved by the persons directory.

**Comment by Candice-Louw** [comment]
FINAL decision - in both windows during the duplicate person search, the following values should be displayed (in this order, please):

First Name | Last Name | Birthdate (Age) | Gender | Telephone Number | District | Community | City | National Health ID | Passport Number

In DE (https://github.com/hzi-braunschweig/SORMAS-Project/issues/4534) and CH contexts, National Health ID and Passport Number are not used and could be removed from this view, however, these are used in the duplicate detection algorithm and should therefore be displayed here - @MateStrysewske - please give input if this is incorrect?

For Telephone Number, use the first entry (assuming multiple phone numbers will be available soon), or if possible, all that are available.

Clean-up of the values that should be used during the duplicate detection process is a separate issue.

**Comment by markusmann-vg** [comment]
First Name | Last Name | Nickname | **Birthdate** (Age) | Present Condition | Gender | **Telephone Number** | District | Community | City | National Health ID | Passport Number

New column for Telephone Number (_primary_) will be added.

Prerequisite: #2744

**Comment by markusmann-vg** [comment]
The data of the found duplicate and the person data should be synched (same colums, same values [like birthdate (age)])

**Comment by MateStrysewske** [comment]
Additional specifications coming from #4745:

* Add zip code to the city column if available
* Add another column containing the street + house number of the person's primary home address

**Comment by maxiheyner** [comment]
Issue Tracker comment

**Comment by StefanKock** [comment]
This ticket is not technically mandatory for 1.58.0, but really desired to be included.
